src/core/clients/Music.ts

```diff
@@ -34,6 +34,7 @@ import {
 } from '../endpoints/index.js';
 
 import { GetSearchSuggestionsEndpoint } from '../endpoints/music/index.js';
+import Account from '../../parser/ytmusic/Account.js';
```

Comment:
Please move this import up to the other ytmusic imports, right at the top above the `Album` one would be great.